.drone.yml

```diff
@@ -492,6 +492,39 @@ steps:
     HOST: grafana-server
   image: cypress/included:9.5.1-node16.14.0-slim-chrome99-ff97
   name: end-to-end-tests-various-suite
+- commands:
```

Comment:
Would these steps be easier to manage in an [image](https://docs.drone.io/pipeline/docker/syntax/images/) we build for this purpose? Besides controlling updates better we could also cut down the boilerplate here. Heck, we could use `cloud-data-sources` as the package repo for such an image.

Reply:
That's a great shout, will implement this as it'll simplify things massively!